src/vsr/replica.zig

```diff
@@ -813,7 +813,11 @@ pub fn ReplicaType(
             });
 
             if (message.header.invalid()) |reason| {
-                log.err("{}: on_message: invalid ({s})", .{ self.replica, reason });
+                log.err("{}: on_message: invalid (command={}, {s})", .{
```

Comment:
I thought you always had to have a thing inside the `{}` if only `{?}`?

Reply:
For printing strings/arrays it is needed (`{s}`, `{any}`.), but for enums/numbers `{}` works just fine.

Reply:
Gotcha, nice.